### ISSUE TITLE
Apps V2: fly launch Doesn't generate an app name when left blank

### ISSUE THREAD

**Issue by jeregrine** [superfly/flyctl#1679]
fly lunch doesn't generate an app name when left blank

* macos 13.2 (22D49)
* `fly version`: fly v0.0.455-pre-1 darwin/arm64 Commit: 97f2c502 BuildDate: 2023-02-07T18:47:54Z

no fly.toml yet

Error 
```text
fly launch
Creating app in <DIR>
Scanning source code
<elixir compile step> 
Detected a Phoenix app
? Choose an app name (leave blank to generate one): (chose none)
automatically selected personal organization: MyName
? Choose a region for deployment: Chicago, Illinois (US) (ord)
Created app  in organization personal
Admin URL: https://fly.io/apps/
Hostname: .fly.dev
Error Could not find App
```

**Comment by jsierles** [comment]
I believe [this line](https://github.com/superfly/flyctl/blob/appsv2/internal/command/launch/launch.go#L287) should be `createdApp.AppName`. This was fixed in the reverted apps2 PR.

**Comment by billyb2** [comment]
This sould be fixed  in #1687
